README.md

```diff
@@ -24,6 +25,33 @@ npm install
 npm link
 ```
 
+## AWS Accounts
+
+This is a multi-account setup, and relies on the ability to assume roles across accounts.
+
+* Accounts
+  * A management account, which is normally called `mgmt`
+  * One or more deployment accounts, which are normally called `dev`, `test`, `uat`, `pre-prod`, `prod`, etc.
+* Roles
+  * Management Role
+  * Deployment Role, with a Trust Relationship to the Management Role (normally called `terraform`)
+
+     ```
+     {
+          "Version": "2012-10-17",
+          "Statement": [
+          {
+               "Effect": "Allow",
+               "Principal": {
+                    "AWS": "arn:aws:iam::<MGMT Account Id>:root"
+               },
+               "Action": "sts:AssumeRole",
+               "Condition": {}
+          }
+     ]
+     }
+     ```
+
```

Comment:
I think something more comprehensive might be useful, for example how to set the app up initially (as the other generators seem to hinge off that working) with a bit of context of what each option is for (as the naming was a little confusing) - especially if the plan is to make this more open source

Reply:
I will add something along those lines to my current branch!